Packages/vcs/Lib/vtk_ui/textbox.py

```diff
@@ -1,40 +1,59 @@
 from text import Label, text_dimensions
-from button import Button
 from datetime import datetime, timedelta
+from line import Line
+import vtk
+
+
+def rotate(point, angle):
+    import math
```

Comment:
I constantly have to adjust for orientation, so I stuck the function to handle that out on its own for ease of access.